### PR TITLE
Add the “How to activate Kinesis log streaming” section.

### PR DESCRIPTION
**Pull request by benguillet**

### CODE REVIEW

File: docs/log_streaming.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+$ emp log -a acme-inc
+Logs are disabled
+```
+
+To activate the log streaming on Empire, you need to set the `EMPIRE_LOG_STREAMER`
+environment variable on your Empire instance(s). Right now the only value supported
+is `kinesis`, but we hope to support more in the future.
+
+When using Amazon Kinesis log streaming, Empire will try to read the logs from the
+Kinesis stream named after the app id. This means that Kinesis streams need to pre-exists
```
Comment (phobologic) [inline docs/log_streaming.md]: Probably worth emphasising that it's the app id (uuid) not the name.

File: docs/log_streaming.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Activating log streaming
+
+By default, log streaming is deactivated on Empire. If you try to run
```
Comment (ejholmes) [inline docs/log_streaming.md]: "on Empire" -> "in Empire"

File: docs/log_streaming.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+`emp log -a <app>`, you will get the following response:
+
+```console
+$ emp log -a acme-inc
+Logs are disabled
+```
+
+To activate the log streaming on Empire, you need to set the `EMPIRE_LOG_STREAMER`
```
Comment (ejholmes) [inline docs/log_streaming.md]: "To activate the log streaming" -> "To Activate log streaming".

File: docs/log_streaming.md (hunk 1 above)
Comment (ejholmes) [inline docs/log_streaming.md]: "This means that the Kinesis streams need to pre-exist..."

File: docs/log_streaming.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+environment variable on your Empire instance(s). Right now the only value supported
+is `kinesis`, but we hope to support more in the future.
+
+When using Amazon Kinesis log streaming, Empire will try to read the logs from the
+Kinesis stream named after the app id. This means that Kinesis streams need to pre-exists
+with logs in them before Empire can forward them to your terminal. We use [logspout-kinesis](https://github.com/remind101/logspout-kinesis) to do so. Our official [Empire AMI](https://github.com/remind101/empire_ami) also take care of running logspout and activating Kinesis log streaming on Empire.
```
Comment (ejholmes) [inline docs/log_streaming.md]: "also takes care"

File: mkdocs.yml [hunk 5]
```diff
@@ -17,6 +17,7 @@ pages:
     - "Process Management - TODO": "process_management.md"
     - "Exposing an app publicly - TODO": "exposing_an_app_publicly.md"
```
Comment (ejholmes) [inline mkdocs.yml]: Side note, I think @bmarini just did this one? We can probably remove the TODO.

**Comment by ejholmes** [comment]
:+1:

File: mkdocs.yml (hunk 5 above)
Comment (benguillet) [inline mkdocs.yml]: it's here #670

**Comment by phobologic** [comment]
:+1:

**Pull request merged**